powershell/tools/index-files.sh

```diff
@@ -4,4 +4,4 @@ if [[ -z "${CODEQL_EXTRACTOR_POWERSHELL_ROOT}" ]]; then
   export CODEQL_EXTRACTOR_POWERSHELL_ROOT="Semmle.Extraction.PowerShell.Standalone.exe"
 fi
 
-"$CODEQL_EXTRACTOR_POWERSHELL_ROOT/tools/$CODEQL_PLATFORM/$CODEQL_POWERSHELL_EXTRACTOR"
\ No newline at end of file
+"$CODEQL_EXTRACTOR_POWERSHELL_ROOT/tools/$CODEQL_PLATFORM/$CODEQL_POWERSHELL_EXTRACTOR" --file-list "%1"
```

Comment:
I know `codeql test run` passes a list of files as an argument here, but what about other commands that call this script? Does this make it always necessary to pass a file list (as opposed to a directory)? My bad for the rookie questions here, just want to ensure I am understanding the extractor as best as possible.

Reply:
I think it always makes sense. For example, this is what Ruby does as well: https://github.com/github/codeql/blob/main/ruby/tools/index-files.cmd. So I think we're good 🤞